src/plan/generational/immix/global.rs

```diff
@@ -129,6 +129,9 @@ impl<VM: VMBinding> Plan for GenImmix<VM> {
         let full_heap = !self.gen.is_current_gc_nursery();
         self.gen.prepare(tls);
         if full_heap {
+            if VM::VMObjectModel::GLOBAL_LOG_BIT_SPEC.is_on_side() {
+                self.immix_space.clear_side_log_bits();
+            }
```

Comment:
We are not calling `self.gen.common.clear_side_log_bits()`, but we do in `StickyImmix::prepare`.  Is this intentional?  I think when using `GenImmix`, we may still directly allocate large objects into LOS and they may become unlogged if they survive a GC.

Reply:
I removed `clear_side_log_bits` in `StickyImmix`. We should deal with log bits for common/base spaces in the same way for the 3 plans, `GenImmix`, `StickyImmix` and `GenCopy`. This should be the same behavior as before the refactoring.